src/migrations/1588256295395_ft_events.ts

```diff
@@ -62,11 +64,11 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
     recipient: 'string',
   });
 
-  pgm.createIndex('ft_events', 'tx_id', { method: 'hash' });
-  pgm.createIndex('ft_events', 'index_block_hash', { method: 'hash' });
-  pgm.createIndex('ft_events', 'microblock_hash', { method: 'hash' });
-  pgm.createIndex('ft_events', 'sender', { method: 'hash' });
-  pgm.createIndex('ft_events', 'recipient', { method: 'hash' });
+  pgm.createIndex('ft_events', 'tx_id', { method: INDEX_METHOD });
+  pgm.createIndex('ft_events', 'index_block_hash', { method: INDEX_METHOD });
+  pgm.createIndex('ft_events', 'microblock_hash', { method: INDEX_METHOD });
+  pgm.createIndex('ft_events', 'sender', { method: INDEX_METHOD });
+  pgm.createIndex('ft_events', 'recipient', { method: INDEX_METHOD });
```

Comment:
Based on our recent conversation, I think we should revert these two fields back to `btree` (non-configurable) and keep only the `bytea` fields (tx_id, *_hash) as optional `hash`/`btree`. Same on all other migration files.

String fields (sender, recipient, contract_id, etc.) would be much more "sortable" in trees than hashes so we'll still have nice performance.